cypress/integration/login.spec.ts

```diff
@@ -19,16 +19,33 @@ describe('Login process', () => {
         cy.get('[data-test="user-avatar"]').should('be.visible');
     });
 
-    it('clicking on user avatar takes you to My Page', () => {
+    it('takes you to My Page when clicking on user avatar', () => {
         cy.visit('/login');
         cy.get('input[aria-label="E-mail address"]').type('testadmin@example.com');
         cy.get('input[aria-label="Password"]').type('password');
         cy.get('input[aria-label="Log in"]').click();
-        cy.get('[data-test="username"').click();
+        cy.get('[data-test="username"]').click();
```

Comment:
Nice catch! This sneaked through the review in your last PR. I'm surprised it was working! This is a good fix, but the same goes here as in my comment above. It should be a separate commit (e.g. along with the change to the message).